addons/io_scene_gltf2/__init__.py

```diff
@@ -1008,7 +1008,7 @@ def invoke(self, context, event):
         for addon_name in preferences.addons.keys():
             try:
                 if hasattr(sys.modules[addon_name], 'glTF2ExportUserExtension') or hasattr(sys.modules[addon_name], 'glTF2ExportUserExtensions'):
-                    exporter_extension_layout_draw[addon_name] = sys.modules[addon_name].draw
+                    exporter_extension_layout_draw[addon_name] = sys.modules[addon_name].draw if hasattr(sys.modules[addon_name], 'draw_export') else sys.modules[addon_name].draw
```

Comment:
Just noticed this line - I think this this should read:

```
exporter_extension_layout_draw[addon_name] = sys.modules[addon_name].draw_export if hasattr(....
```

Reply:
Thanks, fixed